### PR TITLE
Fix urls

### PR DESCRIPTION
**Pull request by gresm**
Update project URLs after creating a fork.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 This is the GitHub organization of Pygame Community, a dedicated online community of pygame fans and users.
 
-We aim to enhance the community around the **[`pygame`](https://www.github.com/pygame/pygame)** Python library by hosting exciting game jams, creating cool community games, and more.
+We aim to enhance the community around the **[`pygame`](https://github.com/pygame-community/pygame-ce)** Python library by hosting exciting game jams, creating cool community games, and more.
 We provide a welcoming space to learn, master and create games with pygame, or to hang out with developers, content creators, gamers, artists, musicians, and a lot of other cool people.
 
-**[Join us on Discord!](https://discord.gg/ZuB2RySPRJ)** | **[YouTube](https://www.youtube.com/channel/UCadZkRAdAfelko4mcD3Uq4Q)** | **[Patreon](https://www.patreon.com/pygamecommunity)**
+**[Join us on Discord!](https://discord.gg/ZuB2RySPRJ)** | **[YouTube](https://www.youtube.com/channel/UCadZkRAdAfelko4mcD3Uq4Q)** | **[Patreon](https://www.patreon.com/pygamecommunity)** | **[Website](https://pyga.me/)**
```
Comment (avaxar) [inline profile/README.md]: I reckon perhaps the website should be placed first?

Comment (gresm) [reply profile/README.md]: Seems reasonable.

**Review by avaxar — APPROVED** [review]
LGTM

**Comment by Mega-JC** [comment]
Hi there, thanks for the PR.

It is well intentioned, but for now we're not currently accepting any PRs on this repository. We'll let people know if that changes.

**Pull request closed without merging**